database/src/test/resources/database.properties.template

```diff
@@ -0,0 +1,8 @@
+# jdbc settings
+test.jdbc.url=jdbc:postgresql://localhost:5432/atum_db
+
+test.jdbc.username=atum_owner
+test.jdbc.password=changeme
+
+# debug - Hint: Use `.only` to run one selected test only. Test are not designed for parallel run.
+test.persist.db=false
```

Comment:
I think that:
* These DB tests can be actually part of the CICD pipelines (as a separated step, so outside of the `sbt test` command that executes all the unit tests we have currently in Scala for this project)
* This config template should not be a config template but an actual config so that the programmatic tests can be automatically run - so either let's store `database.properties` in the repo directly for this module here, or run `cp` command in CICD so that the configuration can be loaded and tests can be automatically executed

Reply:
The CICD and integration tests to be addressed in separate issue and PR.